charts/newrelic-infrastructure/templates/ksm/_naming.tpl

```diff
@@ -0,0 +1,8 @@
+{{- /* Naming helpers*/ -}}
+{{- define "nriKubernetes.ksm.fullname" -}}
+{{- include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" (include "nriKubernetes.naming.fullname" .) "suffix" "ksm") -}}
+{{- end -}}
+
+{{- define "nriKubernetes.ksm.fullname.agent" -}}
+{{- include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" (include "nriKubernetes.naming.fullname" .) "suffix" "agent-ksm") -}}
+{{- end -}}
```

Comment:
I like to see these here, it makes it way more readable than if we had the `truncateToDNSWithSuffix` directly on the yamls!